packages/openapi-fetch/package.json

```diff
@@ -77,6 +77,7 @@
     "openapi-typescript-codegen": "^0.25.0",
     "openapi-typescript-fetch": "^2.0.0",
     "superagent": "^9.0.2",
+    "undici": "^6.14.1",
```

Comment:
See other comment—unfortunately this is a `dependency` because of how it’s used, and I’d like to avoid that. Let’s just copy the `Dispatcher` type locally to prevent this.